model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -482,21 +482,21 @@ class ExcludesTest : WordSpec() {
             "return true if the scope is excluded" {
                 val excludes = Excludes(scopes = listOf(scopeExclude1))
 
-                excludes.isScopeExcluded(scope1) shouldBe true
+                excludes.isScopeExcluded(scope1.name) shouldBe true
```

Comment:
Is it intended that this line is exactly the same as the next ?

Same comment for the changes underneath.

Reply:
This must somehow be crept in during merging. Thanks for spotting, fixed.